packages/lib/jest.config.ts

```diff
@@ -4,12 +4,13 @@ import baseConfig from 'jest-config/base';
 
 const config: Config = {
   ...baseConfig,
-  collectCoverageFrom: ['**/src/**/*.{js,ts}'],
+  collectCoverageFrom: ['**/src/**/*.{js,ts}', '!**/src/**/index.ts'],
```

Comment:
with the `'!**/src/**/index.ts'` pattern, we need to review more careful of what to put under the `index.ts`